backend/codesirius/internal_api/views/problem.py

```
@@ -0,0 +1,40 @@
+logger = logging.getLogger(__name__)
+
+
+class ProblemRetrieveUpdateAPIView(APIView):
+    """
+    This is view is a part of internal API and
+    intended to be used by the internal services only.
+
+    This view is used to retrieve and update a problem.
+    """
+
+    authentication_classes = [APIAuthentication]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)